docs/DeveloperGuide.md

```diff
@@ -133,6 +133,7 @@ The `Model` component,
 
 </div>
 
+Within the `/model` folder, there also exists an `IndexedCsvFile` model that helps with the parsing of CSV files for the Import CSV function. However, this file does not fit within the model component diagram and just serves as a helper model. 
```

Comment:
Please avoid calling it a folder.
```suggestion
Within the `model` package, there also exists an `IndexedCsvFile` model that helps with the parsing of CSV files for the Import CSV function. However, the class does not maintain any persistent instances, and does not fit within the model component diagram, serving solely as a helper model. 
```

Reply:
thanks for the suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/203/commits/3193b02116aeddfa74a438d365ea5575d9acfb0b